### DISCUSSION TITLE
Repository Publicodes

### DISCUSSION THREAD

**Discussion by AdrienRosi** [publicodes/publicodes#460] [Ideas]
Sur le modèle de [Definitely Typed](https://github.com/DefinitelyTyped/DefinitelyTyped), je propose de publier un dépôt Github qui centraliserait les packages Publicodes pour faciliter leur visibilité et leur réutilisation, mais aussi garantir une certaine cohérence dans la couverture du corpus législatif.

Qu'en pensez-vous ?

**Comment by johangirod** [comment]
Une première étape en ce sens est prévu, il est prévu d'avoir une page recensant tous les paquets publicodes sur le nouveau site #489
